docs/source/install.rst

```diff
@@ -0,0 +1,7 @@
+=======
+Install
+=======
+
+Install ComposeML by using :code:`pip`::
```

Comment:
can you add what python versions are supported?

Reply:
Added the supported python version in [this](https://github.com/FeatureLabs/compose-ml/pull/29/commits/dc08c50ee375ebe6532473147c89df8edadda34e#diff-a2e1e3b391e0e24fcd7e517d40d9bba8R5) commit